### PR TITLE
[STINS-003] Implement the main page via proto design

### PR DESCRIPTION
**Pull request by jzha411**
Issue: 
Create the Main page for the Stick insect project.

Solution: 
Created the main page component.

Risk: 
No risk

Reviewed by: 
Donghyun, Lucas

### CODE REVIEW

**Review by aidandayvyd — APPROVED** [review]
LGTM!

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
-import logo from './logo.svg';
+/*import logo from './logo.svg';*/
```
Comment (lucas2005gao) [inline frontend/src/App.js:1]: would be good to just remove this line!

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
+      <header classname = "App-header">
+        <MainPage/>
+      </header>
+      {/* <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <p>
```
Comment (lucas2005gao) [inline frontend/src/App.js:14]: would be good to remove these too! Keep master branch nice and clean

**Review by lucas2005gao — DISMISSED** [review]
image
Tested it locally! Works perfectly. Except we will need to add the page transition logic aye!

**Review by jzha411 — COMMENTED** [review]
Approved of changes

**Comment by jekb2019** [comment]
LGTM

**Review by jekb2019 — APPROVED** [review]

**Pull request merged**